app.js

```diff
@@ -18,22 +18,36 @@ const express = require('express');
 const statusCodes = require('http-status-codes');
 
 const errorTracker = require('./routes/error-tracker');
-const querystring = require('./utils/requests/query-string');
```

Comment:
I think we can delete this module now?

Reply:
It's still used to construct the query string for `querystring.stringify` we log to Stackdriver

Reply:
Delete just `parse` then?

Reply:
Done.